doc/content/guides/connecting-gateways/thethingsgateway/_index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "The Things Gateway"
+description: ""
+weight: 2
+menu:
+  main:
+    weight: 2
+---
+
+The Things Gateway is an MQTT based single channel LoRaWAN gateway. The technical specifications can be found in [the official documentation](https://www.thethingsnetwork.org/docs/gateways/gateway/). This page guides you to connect it to a The Things Stack instance.
+
+
+## Preparation
+
+* User account on a The Things Stack instance with rights to create Gateways and API Keys.
```

Comment:
"a The Things Stack instance" sounds weird. Can we just use "The Things Stack" everywhere?

Reply:
Use numbered lists